website/docs/d/schema_template_contract.html.markdown

```diff
@@ -31,11 +31,21 @@ data "mso_schema_template_contract" "example" {
 ## Attribute Reference ##
 
 * `display_name` - (Read-Only) The name of the Contract as displayed on the MSO UI.
-* `filter_type` - (Read-Only) The type of the Filter. 
+* `filter_type` - (Read-Only) The type of filters of the Contract.
 * `scope` - (Read-Only) The scope of the Contract.
-* `filter_relationships` - A map for the Filter relationship.
+* `target_dscp` - (Read-Only) The dscp value of the Contract.
+* `priority` - (Read-Only) The priority override of the Filter.
+* `filter_relationship` - (Read-Only) A List of Filter relationships.
     * `filter_schema_id` - (Read-Only) The schema ID of the Filter.
     * `filter_template_name` - (Read-Only) The template name of the Filter.
     * `filter_name` - (Read-Only) The name of the Filter.
+    * `filter_type` - (Read-Only) The type of the Filter. 
     * `action` - (Read-Only) The action of the Filter.
     * `directives` - (Read-Only) The directives of the Filter.
+    * `priority` - (Read-Only) The priority override of the Filter.
+
+* `filter_relationships` - (Read-Only) **Deprecated** A map of the Filter relationship.
+    * `filter_schema_id` - (Read-Only) The schema ID of the Filter.
+    * `filter_template_name` - (Read-Only) The template name of the Filter.
+    * `filter_name` - (Read-Only) The name of the Filter.
```

Comment:
Are we removing attributes from the existing filter_relationships?

Reply:
No filter_relationships is the original map that contained 3 attributes schema_id/template_name/name. This is confusing because the naming was wrong. Where originally there was filter_relationships which was a map, then filter_relationship was added after for the list form (https://github.com/CiscoDevNet/terraform-provider-mso/commit/e718007d4f970dd62c93d666fa5d721b24f598a7). So we are not removing we are just not providing any new attributes to the filter_relationships map, since it is deprecated.